jupyter_scheduler/scheduler.py

```diff
@@ -247,16 +257,84 @@ def stop_job(self, job_id):
                         break
 
     def create_job_definition(self, model: CreateJobDefinition) -> str:
-        pass
+        with self.db_session() as session:
+            job_definition = JobDefinition(**model.dict(exclude_none=True))
+            session.add(job_definition)
+            session.commit()
 
-    def update_job_definition(self, model: UpdateJob):
-        pass
+            job_definition_id = job_definition.job_definition_id
+
+        return job_definition_id
```

Comment:
1. Thinking forwards here, it sounds like eventually we'll call `compute_next_run_time()` in the body of this handler, and append it to some kind of queue for later execution. When that job is executed, `compute_next_run_time()` is calculated again and then this process repeats. Is this correct?

2. Sorry if I'm rehashing old discussion, but has there been any proposal to add a "run now" argument to the job definition itself? For example, let's say I create a job definition on Wednesday and want it to run every Monday. I'd like to see the output immediately as well. Are there any plans for this?

Reply:
I was not thinking of a queue at this point but it is possible that we can push the creation of individual jobs to a queue. My initial goal was to save the `next_run_time` at creation and then also with each iteration of the schedule. 
I am not aware of a use case for 2), although users can do this by submitting a run now job and then running a scheduled job after they are satisfied with the results.

Reply:
I think we should eventually have an option that enables a user to run a job definition now. We would probably want a new REST API that takes a `job_definition_id` and overriding options (like `parameters`) and runs it immediately. Can someone open an issue to track that?

Reply:
Done. #89